classes/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] As requested by the previous reviewer to add rentals to `attr_acessor`  to allow access to viewing and adding rentals 👍🏽so kindly update it to be `attr_accessor` instead of `attr_reader` 🔥🙏🏻✅